index.html

```diff
@@ -12,25 +12,39 @@
 		<button class="main_btn goBack">Go back</button>
 	</div>
 	<div class="col-3">
-		<p class="logo"><img src="css/img/kottans.png"></p>
+		<p class="logo"><img src="css/img/kottans.png" alt=""></p>
 	</div>
 	<div class="col-3 sign">
 		<button class="main_btn btn_sgn">Sign In</button>
 	</div>
 </header>
 
 
-<section class="yourOrder">
+<div class="yourOrder">
 	<div class="col-6">
 		<div class="details">
 			<div class="blank"><!--<p><img src="https://png.icons8.com/material/26/6c748d/file.png"></p>--> <p>Your order</p></div>
-			<div>
-				<ul class="list">
+			<div class="list_table">
+				<table>
+					<tr>
+						<th>Items</th>
+						<th>Details</th>
+						<th>Price</th>
+						<th>Delivery</th>
+					</tr>
+					<tr>
+						<td>2 documents</td>
+						<td>2 languages</td>
+						<td>$ 696.06</td>
+						<td>48 hours*</td>
+					</tr>
+				</table>
+				<!-- <ul class="list">
 					<li><span>Items</span></br></br>2 documents</li>
 					<li><span>Details</span></br></br>2 languages</li>
 					<li><span>Price</span></br></br>$ 696.06</li>
 					<li><span>Delivery</span></br></br>48 hours*</li>
-				</ul>
+				</ul> -->
```

Comment:
How about using [`address`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address) element?
Or at least use anchor element (`<a>`) with href="mailto:help@qtest.com". check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) under *Creating an email link* and *Creating phone link*